src/overlays/actors/ovl_En_Insect/z_en_insect.c

```diff
@@ -10,30 +10,39 @@
 
 #define FLAGS 0
 
+#define IS_DROPPED(type) ((type) == INSECT_TYPE_FIRST_DROPPED || (type) == INSECT_TYPE_EXTRA_DROPPED)
+
 void EnInsect_Init(Actor* thisx, GlobalContext* globalCtx);
 void EnInsect_Destroy(Actor* thisx, GlobalContext* globalCtx);
 void EnInsect_Update(Actor* thisx, GlobalContext* globalCtx);
 void EnInsect_Draw(Actor* thisx, GlobalContext* globalCtx);
 
-void func_80A7C3A0(EnInsect* this);
-void func_80A7C3F4(EnInsect* this, GlobalContext* globalCtx);
-void func_80A7C598(EnInsect* this);
-void func_80A7C5EC(EnInsect* this, GlobalContext* globalCtx);
-void func_80A7C818(EnInsect* this);
-void func_80A7C86C(EnInsect* this, GlobalContext* globalCtx);
-void func_80A7CAD0(EnInsect* this, GlobalContext* globalCtx);
-void func_80A7CBC8(EnInsect* this);
-void func_80A7CC3C(EnInsect* this, GlobalContext* globalCtx);
-void func_80A7CE60(EnInsect* this);
-void func_80A7CEC0(EnInsect* this, GlobalContext* globalCtx);
-void func_80A7D1F4(EnInsect* this);
-void func_80A7D26C(EnInsect* this, GlobalContext* globalCtx);
-void func_80A7D39C(EnInsect* this);
-void func_80A7D460(EnInsect* this, GlobalContext* globalCtx);
+void EnInsect_SetupSlowDown(EnInsect* this);
+void EnInsect_SlowDown(EnInsect* this, GlobalContext* globalCtx);
+void EnInsect_SetupCrawl(EnInsect* this);
+void EnInsect_Crawl(EnInsect* this, GlobalContext* globalCtx);
+void EnInsect_SetupRunFromPlayer(EnInsect* this);
+void EnInsect_RunFromPlayer(EnInsect* this, GlobalContext* globalCtx);
+void EnInsect_SetupCaught(EnInsect* this);
+void EnInsect_Caught(EnInsect* this, GlobalContext* globalCtx);
+void EnInsect_SetupDig(EnInsect* this);
+void EnInsect_Dig(EnInsect* this, GlobalContext* globalCtx);
+void EnInsect_SetupWalkOnWater(EnInsect* this);
+void EnInsect_WalkOnWater(EnInsect* this, GlobalContext* globalCtx);
+void EnInsect_SetupDrown(EnInsect* this);
+void EnInsect_Drown(EnInsect* this, GlobalContext* globalCtx);
+void EnInsect_SetupDropped(EnInsect* this);
+void EnInsect_Dropped(EnInsect* this, GlobalContext* globalCtx);
```

Comment:
@mzxrules didn't like "Dropped" but idk what to rename this to

Reply:
`EnInsect_Released`?

Reply:
@mzxrules please review if you still think this should be changed, I'll mark these as resolved